docs/issue-labels.rst

```diff
@@ -7,93 +7,89 @@ what they stand for.
 .. _GitHub issue tracker: https://github.com/rtfd/readthedocs.org/issues
 
 *Bug*
-    An issue describing unexpected or malicious behaviour of the
-    readthedocs.org software.
+    An issue describing unexpected or malicious behaviour of the readthedocs.org
+    software. A Bug issue differs from an Improvement issue in that Bug issues
+    are given priority on our roadmap. On release, these issues generally only
+    warrant incrementing the patch level version.
```

Comment:
Do we need a distinction between Bugs & Accepted Bugs, similar to Features/Improvements? It feels like we'd want to have some kind of two-step there as well.

Reply:
I'd be +1 on this